### DISCUSSION TITLE
밀린 기술부채 정리하기

### DISCUSSION THREAD

**Discussion by amaran-th** [BDD-CLUB/challenge-every-week-posting#99] [4. 기타 글 모음]
원래 저번 달 분량인데 계속 밀려서 이제야 정리를 했네요...ㅋㅋ
레벨2 7주차 때 학습하면서 생긴 궁금증/처음 알게된 용어를 정리해봤습니다!
[게시글 링크](https://amaran-th.github.io/%EC%9A%B0%ED%85%8C%EC%BD%94%205%EA%B8%B0/[%EA%B8%B0%EC%88%A0%EB%B6%80%EC%B1%84]%20%EC%9A%B0%ED%85%8C%EC%BD%94%205%EA%B8%B0%20%EB%A0%88%EB%B2%A82%20-%207%EC%A3%BC%EC%B0%A8/)

**Comment by shkisme** [comment]
되게 폭넓게 공부하고 계시네요! 잘 읽었습니다!

---

<img width="813" alt="image" src="https://github.com/amaran-th/every-week-posting-challenge/assets/92802207/e0b61a81-83c3-41d1-9326-90a036ead011">

이 부분 위 아래 예제 코드가 똑같은 것 같아용
